refinery/data_set_manager/models.py

```diff
@@ -595,17 +595,16 @@ def run_generate_auxiliary_node_task(self):
         file_store_item = self.get_file_store_item()
 
         # Check if we pass the logic to generate aux. Files/Nodes
-        if (file_store_item.filetype.used_for_visualization and
-            file_store_item.is_local() and
+        if (file_store_item and file_store_item.filetype and
```

Comment:
very lengthy, helper method?

Reply:
I think the whole `run_generate_auxiliary_node_task()` might need refactoring or rewriting. This is just a quick fix to make it more robust.